tests/test_dsp.py

```diff
@@ -583,6 +583,142 @@ def test_minimum_phase_multidim():
     npt.assert_allclose(imp_minphase.time, imp_zerophase.time, atol=1e-10)
 
 
+def test_delay_ir():
```

Comment:
I'd prefer to stick with the test names incl. the whole function name to easily find it :)
```suggestion
def test_impulse_response_delay():
```
Same holds for tests below